addons/website_sale/models/product.py

```diff
@@ -17,7 +19,23 @@ class ProductPricelist(models.Model):
     _inherit = "product.pricelist"
 
     def _default_website(self):
-        return self.env['website'].search([], limit=1)
+        """ Find the first company's website, if there is one. """
+        company_id = self.env.user.company_id.id
+        if self.pool._init:
+            # During l10n modules install, it will change the company currency
+            # that will create a default pricelist for that currency.
+            # We should not search for a website in that case to avoid
+            # `_check_websites_in_company` constraint.
+            return self.env['website']
```

Comment:
no more needed if we move constraint in master

but always not understand this case.
It is louchhhh to make special case, to bypass a new constraint...

Reply:
Constraint about misconfiguration on multi-website/multi-company and its related `if` on `_default_website` have been removed from this PR and will be added back in master in another PR.